Orange/data/sql/backend/mssql.py

```diff
@@ -9,6 +9,13 @@
 from Orange.data.sql.backend.base import ToSql, BackendError
 
 
+def parse_ex(ex):
+    try:
+        return ex.args[0][1].decode().split("\n")[-2]
```

Comment:
You could use `splitlines()` instead of `split("\n")`. Supposedly safer. Change if you'll be changing anything anyway; leave as it is if not.